latex/chain.tex

```diff
@@ -256,8 +256,13 @@ \subsection{New Epoch Transition}
         \vdash
         \var{es'}\trans{epoch}{\var{e}}\var{es''}
       }
-      \\
-      \var{pd'} \leteq \mathsf{TODO}\text{ (calculate the new pool distr from }es''\text{)}
+      \\~\\~\\
+      {\begin{array}{r@{~\leteq~}l}
```

Comment:
for the format of the definition of `applyRUp`: to be honest, I would prefer this in a different format. Less than half of the entries are changed, maybe something like Haskell record update syntax would be ok?

Reply:
I quite like the syntax for `applyRUp` :-)

Reply:
yea, it's a bit weird that this is the only place we do this kind of thing. I like that visually it's really clear what is going on and what is changing. Originally I tried to do it with lots of let bindings and pattern matching, but I felt it was far from clear... Record update syntax would work ok, for the deltas we need to dig out the original values, so I don't think it would be very compact.  we need lenses... :grimacing:

Reply:
Same here. I like the `applyRUp` syntax. It is true that some entries remain unchanged, but it makes things easier to read.

Reply:
and yea, `e` should be `e_l+1`. perhaps that is better. as it is now, we would still be fine if we didn't get a block until several epochs later, but in practice that would be bad.